src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts

```diff
@@ -93,4 +88,127 @@ export const EDIT_PAGE_CSS = `
+        background-repeat: no-repeat;
+        background-position: 4px;
+        background-color:#0E80CB;
+    }
+    
+     .dotedit-container__add:hover{
+        background-color:#0b629b;
+     }
+     
+     .dotedit-container__add:focus{
+        background-color: #0b629b
+     }
+     
+     .dotedit-container__add:active{
+        background-color: #07446c;
+     }
+     
+      .dotedit-contentlet__drag {
+        touch-action: none;
+        cursor: move;
+        background-repeat: no-repeat;
+        background-position: 7px;
+        background-image: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSIjNDQ0NDQ0IiBoZWlnaHQ9IjE4IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+ICAgIDxkZWZzPiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgwVjB6IiBpZD0iYSIvPiAgICA8L2RlZnM+ICAgIDxjbGlwUGF0aCBpZD0iYiI+ICAgICAgICA8dXNlIG92ZXJmbG93PSJ2aXNpYmxlIiB4bGluazpocmVmPSIjYSIvPiAgICA8L2NsaXBQYXRoPiAgICA8cGF0aCBjbGlwLXBhdGg9InVybCgjYikiIGQ9Ik0yMCA5SDR2MmgxNlY5ek00IDE1aDE2di0ySDR2MnoiLz48L3N2Zz4=);
+    }
+     
+     .dotedit-contentlet__edit{
+        background-image: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSIjNDQ0NDQ0IiBoZWlnaHQ9IjE4IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4gICAgPHBhdGggZD0iTTMgMTcuMjVWMjFoMy43NUwxNy44MSA5Ljk0bC0zLjc1LTMuNzVMMyAxNy4yNXpNMjAuNzEgNy4wNGMuMzktLjM5LjM5LTEuMDIgMC0xLjQxbC0yLjM0LTIuMzRjLS4zOS0uMzktMS4wMi0uMzktMS40MSAwbC0xLjgzIDEuODMgMy43NSAzLjc1IDEuODMtMS44M3oiLz4gICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjwvc3ZnPg==);
+        background-repeat: no-repeat;
+        background-position: 7px;
+     }
+     
+     .dotedit-contentlet__remove{
+        background-image: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSIjNDQ0NDQ0IiBoZWlnaHQ9IjE4IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4gICAgPHBhdGggZD0iTTE5IDYuNDFMMTcuNTkgNSAxMiAxMC41OSA2LjQxIDUgNSA2LjQxIDEwLjU5IDEyIDUgMTcuNTkgNi40MSAxOSAxMiAxMy40MSAxNy41OSAxOSAxOSAxNy41OSAxMy40MSAxMnoiLz4gICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjwvc3ZnPg==);
+        background-repeat: no-repeat;
+        background-position: 7px;
+     }
+          
+     .dotedit-container__menu {
+          width: 100px;
+          background-color: #ffffff;
+          box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.2);
+          font-family:Roboto, "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+          font-size: 13px;
+          position: absolute;
+          z-index:1003;
+          visibility: hidden;
+          opacity: 0;
+          transition: 200ms linear;
```

Comment:
We are using ```200ms linear``` a lot... I think we can save that in a var and use it in this string template.

The same goes with the box-shadows, check the ```$md-shadows``` vars in ```scss``` I think we can use variables as well here because at the end of the day is just a JS string.